aws.yaml

```diff
@@ -1,701 +1,537 @@
 - active: false
   authorities:
-    - endpoint: http://app-operator:8000
-      name: app-operator
-      version: 1.0.0
-    - endpoint: http://aws-operator:8000
-      name: aws-operator
-      version: 5.5.0
-    - endpoint: http://cert-operator:8000
-      name: cert-operator
-      version: 0.1.0
-    - endpoint: http://chart-operator:8000
-      name: chart-operator
-      version: 0.7.0
-    - endpoint: http://cluster-operator:8000
-      name: cluster-operator
-      provider: aws
-      version: 0.21.0
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 5.5.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: aws
+    version: 0.21.0
```

Comment:
This is personal taste thing but it's easier for me to read when lists are indented in yaml so if you don't mind I would rather have them indented like they were.

Reply:
We should also split PRs for this. Dropping endpoints is one PR. Changing formatting is another PR. This makes the process and bringing changes forth way easier.

Reply:
I didn't intended to change the indentation, this was the result of the command. Closing here, and will re-work this later on.